lib/index.js

```diff
@@ -1385,4 +1385,34 @@ class Hatch {
 
 exports.Hatch = Hatch;
 
+
+/**
+ * Get parsed settings to limit the ingested items.
+ *
+ * The `max_items` and `max_days_old` parameters can be overridden by the
+ * `LIBINGESTER_MAX_ITEMS` and `LIBINGESTER_MAX_DAYS_OLD` environment
+ * variables, respectively.
+ *
+ * @param {number} [max_items=Infinity] - Maximum number of entries to ingest
+ * @param {number} [max_days_old=1] - Maximum age of entries to ingest, in days
+ * @returns {Object} - An object with parsed settings `max_items` and `oldest_date`
+ */
+function get_max_settings(max_items=Infinity, max_days_old=1) {
```

Comment:
`get_max_settings()` is a pretty awkward name... I would rename this to e.g. `get_ingestion_settings()`, or `get_ingestion_limits()`. Also it may be worth moving it to the util section.